packages/desktop-client/src/components/budget/rollover/RolloverComponents.tsx

```diff
@@ -350,7 +372,10 @@ export const ExpenseCategoryMonth = memo(function ExpenseCategoryMonth({
   );
 });
 
-export function IncomeGroupMonth() {
+type IncomeGroupMonthProps = {
+  month: string;
```

Comment:
<!-- Suggestions are specific requests to improve the subject under review. It is assumed that we all want to do what's best, so these comments are never dismissed as “mere suggestions”, but are taken seriously.  -->
**💬 suggestion:** ‏
```suggestion
  selectedMonth: string;
```

Reply:
See comment above.